composer.json

```diff
@@ -10,13 +10,13 @@
   ],
   "minimum-stability": "dev",
   "require": {
-    "php": "^7.1.3",
-    "illuminate/notifications": "~5.8.0|^6.0|^7.0|^8.0",
+    "php": "^7.1.3|^8.0",
+    "illuminate/notifications": "~5.8.0|^6.0|^7.0|^8.0|^8.20",
     "aws/aws-sdk-php": "^3.110"
   },
   "require-dev": {
-    "mockery/mockery": "^1.0",
-    "phpunit/phpunit": "^7.0|^8.0"
+    "mockery/mockery": "^1.0|^1.4",
```

Comment:
Same question as above...I can see adding 9.0 for PHPUnit below but wouldn't `^1.0` cover `^1.4` as well?

Reply:
I thought like `~` operator.  Now i read carefully difference between `~` and `^`. Then I saw you are right, this additions are unnecessary.
I deleted them and tested, now works perfectly again.  
Thanks for your review 👍🏽